bot.py

```diff
@@ -154,6 +154,23 @@ def expiry_message() -> str:
     return "\nThis link will expire in %d minutes" % config["default_ttl"]
 
 
+@client.listen('on_message')
+async def unhandled_dm(message):
+    """
+    When a DM is received and it didn't match any known commands, show the help prompt.
```

Comment:
isnt this already being done automatically?

Reply:
No because discord.py is a dumb poopy head and UnknownCommandError only happens if there's a prefix match and then the command doesn't exist

Reply:
ohhhh i misunderstood what it was doing